apps/src/blockly/addons/functionEditor.js

```diff
@@ -87,10 +89,12 @@ export default class FunctionEditor {
       .getElementById(MODAL_EDITOR_CLOSE_ID)
       .addEventListener('click', () => this.hide());
 
-    // Delete handler
-    document
-      .getElementById(MODAL_EDITOR_DELETE_ID)
-      .addEventListener('click', this.onDeletePressed.bind(this));
+    // Delete handler. We only enable the delete button for writeable workspaces.
```

Comment:
Nit: I read "Delete handler" as "We want to delete the handler". Could we say "Handler for delete button" or similar instead?

Reply:
yep, will update